examples/routing_service/routing_service_file_adapter/cpp/README.md

```diff
@@ -43,6 +43,10 @@ $cmake -DCONNEXTDDS_DIR=<Connext DDS Directory>
 cmake --build .
 ```
 
+**Note**: when compiling on a Windows 64-bit machine you will need to add the
+`-A x64` parameter to the call to CMake.
+
```

Comment:
I think markdownlint will complain about blank lines. Take a look at the results

Reply:
Yes, I have been fixing them, there should be two other commits.

Reply:
@fgaranda , they seem fixed now, so I think if you approve we're ready to merge.